html/elements/summary.json

```diff
@@ -54,16 +54,16 @@
             "description": "<code>display: list-item</code>",
             "support": {
               "chrome": {
-                "version_added": false,
-                "notes": "Chrome currently doesn't use <code>display: list-item</code> on the <code>&lt;summary&gt;</code> element, so <code>display: block</code> will not automatically hide the disclosure widget. Instead, use the non-standard pseudo-element <code>::-webkit-details-marker</code> to change the disclosure widget in Chrome. See <a href='https://crbug.com/590014'>bug 590014</a> for details."
+                "version_added": "89",
+                "notes": "<code>::-webkit-details-marker</code> pseudo element selector is deprecated. Please use <code>::marker</code> instead."
```

Comment:
I propose that we drop this note, now the feature is supported. This would apply to this and all the browsers with this note.

That said, we ought to track support for the non-standard selector. I've recorded that problem in https://github.com/mdn/browser-compat-data/issues/9869.

Reply:
Good idea